podcast/2.5.md

```diff
@@ -0,0 +1,455 @@
+---
+title: 'Christopher Chedeau, co-creator of React Native and Prettier'
+publishDate: 2021-08-31T10:00-07:00
+tags: [podcast]
+slug: christopher-chedeau
+published: true
+---
+
+<!-- START AUDIO -->
+
+<audio className="object-center" src="https://www.buzzsprout.com/1097978/9111652-christopher-chedeau-excalidraw.mp3" controls preload="none"></audio>
+
+<!-- END AUDIO -->
+
+<!-- START GUESTS -->
+<span>
+Christopher Chedeau, Beyang Liu
+</span>
+<!-- END GUESTS -->
+
+<!-- START SUMMARY -->
+
+On the eve of the pandemic, Christopher Chedeau was procrastinating on performance reviews at Facebook and decided to hack together a simple drawing app. That weekend project became Excalidraw, an open source virtual whiteboard so popular that its users have basically demanded a startup form around it so that they can bring it to work. Christopher tells the backstory of Excalidraw's meteoric rise, as well as his story of joining Facebook and coming to America, and tales from the early days of React and how React Native was born.
+
+<!--END SUMMARY -->
```

Comment:
It's a tiny thing, but there's a missing space in line 25 - it should be `<!-- END SUMMARY -->`. You can see the exact formatting needed here: https://github.com/sourcegraph/about/blob/main/website/src/pages/podcast.tsx#L37

Reply:
🤦‍♀️

Reply:
That is probably my bad! Thanks for catching, @csbailey5t!

Reply:
```suggestion
<!-- START SUMMARY -->

On the eve of the pandemic, Christopher Chedeau was procrastinating on performance reviews at Facebook and decided to hack together a simple drawing app. That weekend project became Excalidraw, an open source virtual whiteboard so popular that its users have basically demanded a startup form around it so that they can bring it to work. Christopher tells the backstory of Excalidraw's meteoric rise, as well as his story of joining Facebook and coming to America, and tales from the early days of React and how React Native was born.

<!-- END SUMMARY -->
```